satinfo/amsua_metop-c/readme

```diff
@@ -1 +1 @@
-metop-c  20191107 060000  21001231 240000  amsua 14  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15     # from operational satinfo
+metop-c  20191107 000000  21001231 240000  amsua 14  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
```

Comment:
Jeff the start date in notes is "2019**09**17" yours is "2019**11**07" I think you misread it. Or i misread it.

Reply:
fixed